bdd/features/e2e/E2E-010-cli.feature

```diff
@@ -15,13 +15,13 @@ This feature checks CLI functionalities
         Then I get Hub load information
         And host is still running
 
-    @ci-api @cli
+    @remove
```

Comment:
why `remove`? There are some sample sequences in `bdd/data/sequences` directory we can use to test `pack`